lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -1566,9 +1568,24 @@ LogicalResult FIRRTLLowering::visitDecl(WireOp op) {
   if (resultType.isInteger(0))
     return setLowering(op, Value());
 
-  // Name attr is requires on sv.wire but optional on firrtl.wire.
+  // Name attr is required on sv.wire but optional on firrtl.wire.
   auto nameAttr = op.nameAttr() ? op.nameAttr() : builder.getStringAttr("");
-  return setLoweringTo<sv::WireOp>(op, resultType, nameAttr);
+
+  // Prepend the name of the module to make the symbol name unique in the symbol
+  // table, it is already unique in the module. Checking if the name is unique
+  // in the SymbolTable is non-trivial.
+  auto symName =
+      op.nameAttr()
+          ? builder.getStringAttr("__" +
+                                  cast<hw::HWModuleOp>(op->getParentOp())
+                                      .getNameAttr()
+                                      .getValue()
+                                      .str() +
+                                  "__" + nameAttr.getValue().str())
+          : StringAttr();
```

Comment:
Please change his into an if statement to fix the crazy formatting:

```
 auto symName = op.nameAttr();
 if (symName) {
   auto moduleName = cast<hw::HWModuleOp>(op->getParentOp()).getName();
   symName = stuff;
}
```

Reply:
Fixed the formatting.